sdk/identity/azure-identity/inc/azure/identity/client_certificate_credential.hpp

```diff
@@ -114,7 +124,25 @@ namespace Azure { namespace Identity {
         = Core::Credentials::TokenCredentialOptions());
 
     /**
-     * @brief Constructs a Client Secret Credential.
+     * @brief Constructs a Client Certificate Credential.
+     *
+     * @param tenantId Tenant ID.
+     * @param clientId Client ID.
+     * @param clientCertificate The x509 certificate which is used for signing, in base64 string
```

Comment:
What is the format of the X509 certificate? There are several possible formats for X.509 certificates, which one is this encoded in? 

Also, why are you passing a base64 encoded string as a `std::vector<uint8_t>`? The entire reason for base64 encoding a value is to enable passing it in as a string. If you're going to take the value as an array of bytes, you should probably pass the certificate without base64 encoding it.

Reply:
`std::string` for base64 strings rather than `vector<uint8_t>` makes perfect sense. Will update

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-cpp/pull/5991